# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/web-animations/__dir__.ini
# Commit: 6f62586a139b
# Full Hash: 6f62586a139b1fcc4817dddd924400958d89a6bb
# Author: Zach Hoffman <zach@zrhoffman.net>
# Date: 2023-12-18 17:17:57
# Regressor Bug: 1846516
# File Overlap Count: 2
# Description:
#   Bug 1846516 - [css-properties-values-api] Animate custom properties in a discrete way. r=firefox-style-system-reviewers,emilio
#   
#   This patch also updates the bug ID for a FIXME leftover from bug 1840478
#   to bug 1869476, since the same FIXME is added in D190758.
#   
# ==============================================================================

diff -r 729abedf9bda -r 6f62586a139b testing/web-platform/meta/web-animations/__dir__.ini
--- a/testing/web-platform/meta/web-animations/__dir__.ini	Mon Dec 18 09:24:08 2023 +0000
+++ b/testing/web-platform/meta/web-animations/__dir__.ini	Mon Dec 18 09:24:08 2023 +0000
@@ -1,1 +1,1 @@
-prefs: [dom.animations-api.compositing.enabled:true, dom.animations-api.getAnimations.enabled:true, dom.animations-api.timelines.enabled:true, layout.css.marker.restricted:false]
+prefs: [dom.animations-api.compositing.enabled:true, dom.animations-api.getAnimations.enabled:true, dom.animations-api.timelines.enabled:true, layout.css.marker.restricted:false, layout.css.properties-and-values.enabled:true]
